content/en/watchdog/faulty_cloud_saas_api_detection.md

```diff
@@ -26,9 +26,78 @@ Whenever a faulty deployment is detected, Watchdog creates an event in the [Even
 2. Choose **Watchdog**.
 3. Select `Third Party` in the alert category.
 
+
+## Supported providers
+Watchdog monitors the status of the following external providers' APIs: 
+
+| External Provider | API monitored |  
+|----------|--------------------|
+| Amplitude | api.amplitude.com |
+| Atlassian | *.atlassian.net |
+| Auth0 | *.auth0.com |
+| Binance | api.binance.com     |
+| Braintree | api.braintreegateway.com |
+| Coreweave | *.coreweave.com |
+| Cloudflare | api.cloudflare.com |
+| Confluent | api.confluent.cloud & api.telemetry.confluent.cloud |
+| Envoy | api.envoy.com |
+| Facebook | graph.facebook.com |
+| Github | api.github.com |
+| Google | developers.google.com |
+| Hubspot | api.hubspot.com |
+| Intercom | api.intercom.io |
+| Mapbox | api.mapbox.com |
+| Mixpanel | api.mixpanel.com |
+| OpenAI | *.openai.com|
+| Pagerduty | api.pagerduty.com |
+| Palo Alto Networks | api.urlcloud.paloaltonetworks.com |
+| Render | api.render.com |
+| Sendgrid | .sendgrid.com |
+| Slack | *.slack.com |
+| Soundcloud | api.soundcloud.com |
+| Splunk | *.splunkcloud.com |
+| Square | connect.squareup.com |
+| Stripe | api.stripe.com |
+| Towerdata | api.towerdata.com |
+| Twilio | api.twilio.com |
+| Twitter | api.twitter.com |
+| Zendesk | *.zendesk.com |
+
+The following AWS services are monitored (.*amazonaws.com):
```

Comment:
Doublechecking because it doesn't look like the other APIs:

```suggestion
The following AWS services are monitored (*.amazonaws.com):
```

Reply:
thanks for checking
this is expected here